windows_integration/security_package/src/protocol/v1.rs

```diff
@@ -0,0 +1,95 @@
+use kanidm_windows::secpkg::ap_proto::v1::*;
+use tracing::{event, Level};
+
+use crate::package::KANIDM_WINDOWS_CLIENT;
+
+pub async fn handle_request(request: &AuthPkgRequest) -> AuthPkgResponse {
+    return match request {
```

Comment:
@Water261 I think that the best way to proceed here will be to simply try running the module like this and attempt a login and see what breaks during usage. We can "guess" what we may need to implement here, but knowing exactly what windows actually calls would be the best way to proceed and implement things. That will also let us know if this is all connected up correctly too. When it comes to SID generation, I have an algorithm in mind too :)